scrapy/utils/reactor.py

```diff
@@ -90,11 +90,16 @@ def set_asyncio_event_loop(event_loop_path):
         asyncio.set_event_loop(event_loop)
     else:
         try:
+            asyncio.get_running_loop()
             event_loop = policy.get_event_loop()
         except RuntimeError:
-            # `get_event_loop` is expected to fail when called from a new thread
+            # Raised by policy.get_running_loop() when called from a new thread
             # with no asyncio event loop yet installed. Such is the case when
-            # called from `scrapy shell`
+            # called from `scrapy shell`.
+            # In Python 3.10, but not in 3.11, policy.get_event_loop() seems to
+            # log a warning when called from the main thread, so we use
+            # asyncio.get_running_loop() to get an earlier RuntimeError as a
+            # workaround.
```

Comment:
I think this is happening, it feels like a Python 3.10 bug, but I have found no information about it.

If you comment out `asyncio.get_running_loop()` above, `test_install_asyncio_reactor` fails in Python 3.10 but _not_ in Python 3.11. You can try for yourself with:

```
tox -e py311,py310 -- tests/test_utils_asyncio.py::AsyncioTest tests/test_command_shell.py::ShellTest::test_shell_fetch_async
```

Reply:
My Python 3.11 is 3.11.0, that explains the 3.10-3.11 difference I get locally.